pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java

```diff
@@ -212,10 +220,6 @@ void runCmd() throws Exception {
             }
             print("Updated successfully");
         }
-
-        protected void validateSinkConfigs(SinkConfig sinkConfig) {
```

Comment:
Why are we removing this? The validation logic for update sink is different than that of create sink. By removing this logic, the validation would default to the one that is run for creating sinks which require more fields to be specified than an update.

Reply:
I missed that difference, thanks for point out this! Changed

Reply:
@jerrypeng Can you help to point out the difference between update and create validation? I found that both of them called `sinkConfig.getArchive()`, so I think the validation logic for update sink should do the `archive` validation as the create sink, otherwise it will still throw the NPE (path is 'null')

Reply:
for create requests, all parameters that are required need to be set. However for update, only a subset will do.